File: Day-6/0.c
#include <stdio.h>
#include <string.h>
#include <math.h>

int main() {
    int i,j;
    char s[10000];
    scanf("%s",s);
    int c=sqrt(strlen(s));
    c+=1;
    for(i=0;i<c;i++){
        for(j=i;j<strlen(s);j+=c){
            printf("%c",s[j]);
        }
        printf(" ");
    }
    return 0;
}

File: Day-6/0.py
import math
s=input()
s=s.replace(" ","")
r=int(math.sqrt(len(s)))
c=r+1
for i in range(0,c):
    for j in range(i,len(s),c):
        print(s[j],end='')
    print(" ",end='')
